local-lattes-backend/cafeRecAPI/Data/CafeDBContext.cs

```diff
@@ -0,0 +1,51 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Hosting;
+using cafeRecAPI.Models;
+using static cafeRecAPI.Models.CafeLocation;
+namespace cafeRecAPI.Data
+{
+	public class CafeDBContext : DbContext
+	{
+        public CafeDBContext(DbContextOptions<CafeDBContext> options) : base(options) { }
+
+        public DbSet<Review> Reviews { get; set; }
+        public DbSet<Cafe> Cafes { get; set; }
+        public DbSet<Tag> Tags { get; set; }
+        public DbSet<SearchLocation> Locations { get; set; }
+        public DbSet<CafeSearchLocation> CafeSearchLocations { get; set; }
+
+        protected override void OnModelCreating(ModelBuilder modelBuilder)
```

Comment:
Weird naming OnCafeCreated` would be better. Don't name variable/functions with "-ing" what does that mean? Does it mean it's created or it's creating? and if it's "creating" when is it finished created.

Reply:
it's the rules for any model relationship, not just cafe, unsure what else to name it